app/src/main/res/values/component_colors.xml

```diff
@@ -22,6 +22,7 @@
   <color name="component_color_shared_previous_responses_header_item_container_background_color">@color/color_palette_highlighted_text_color</color>
   <color name="component_color_shared_transparent_background_color">@color/color_palette_transparent_background_color</color>
   <color name="component_color_shared_white_background_color">@color/color_palette_shared_white_background_color</color>
+  <color name="component_color_shared_secondary_button_background_color">@color/color_palette_secondary_button_background_color</color>
```

Comment:
We already have `color_palette_primary_button_background_color` so we not need `component_color_shared_secondary_button_background_color` extra color. 
What you think @BenHenning ?

Reply:
Yes looks good. We keep these two separate.